src/main/java/Main.java

```diff
@@ -1,6 +1,75 @@
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        int visitorCount = getVisitorCount();
+        Calculator calculator = calculateGoods(visitorCount);
```

Comment:
Так как переменная `calculator` после вызова функции `calculateGoods` не используется, то её можно убрать, вызов функции от этого не изменится
```suggestion
        calculateGoods(visitorCount);
```

Reply:
Исправил